rules/windows/defense_evasion_unusual_system_vp_child_program.toml

```diff
@@ -31,7 +31,7 @@ type = "eql"
 
 query = '''
 process where host.os.type == "windows" and event.type == "start" and
-  process.parent.pid == 4 and
+  process.parent.pid == 4 and process.executable: "?*" and
```

Comment:
Just want to check that process.executable is not empty? Wouldn't `process.executable != null` work there?

Reply:
> The field != null syntax expects that the field doesn't exist/is not populated in the event

Ref: https://github.com/elastic/sdh-security-team/issues/462#issuecomment-1274562038

![195080713-6ef33182-cc22-4f39-bca6-b63639d0009f](https://github.com/elastic/detection-rules/assets/26856693/a7cb43e7-95c6-4200-99f6-c05edd3dfdc4)